.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[registries.crates-io]
+protocol = "sparse"
```

Comment:
I had to upgrade to a newer version of Rust to make this work. Should we specify the minimum rust version in `Cargo.toml`? 

https://blog.rust-lang.org/2021/10/21/Rust-1.56.0.html#cargo-rust-version

Reply:
Updated. We always target the latest, but good call that specifying a minimum is the right move.